prow/config/config.go

```diff
@@ -470,7 +470,17 @@ func (c *Config) GetPresubmits(gc git.ClientFactory, identifier string, baseSHAG
 
 // GetPresubmitsStatic will return presubmits for the given identifier that are versioned inside the tested repo.
 func (c *Config) GetPresubmitsStatic(identifier string) []Presubmit {
-	return c.PresubmitsStatic[identifier]
+	keys := []string{identifier}
```

Comment:
Nit: IIUC this function just converts "https://org/repo" into a 2-element list of "https://org/repo" and "org/repo", assuming "https://org/repo" is a Gerrit repo (`IsGerritOrg`).

How about just writing a generic function that performs this creation of a 2-element list, and then similarly do a lookup of either `[]Presubmit` or `[]Postsubmit` for it? I think that way you could deduplicate the logic in here with the lines 503-513.

Reply:
ack. The lookup of prowjobs are centralized in this PR, not foreseeing this function being used in the future. However will keep this in mind if it occurred in the future